src/databricks/labs/ucx/install.py

```diff
@@ -486,9 +496,12 @@ def _upload_wheel(self) -> str:
             remote_wheel = f"{self._install_folder}/wheels/{local_wheel.name}"
             remote_dirname = os.path.dirname(remote_wheel)
             with local_wheel.open("rb") as f:
-                self._ws.dbfs.mkdirs(remote_dirname)
-                logger.info(f"Uploading wheel to dbfs:{remote_wheel}")
-                self._ws.dbfs.upload(remote_wheel, f, overwrite=True)
+                try:
+                    self._ws.dbfs.mkdirs(remote_dirname)
+                    logger.info(f"Uploading wheel to dbfs:{remote_wheel}")
+                    self._ws.dbfs.upload(remote_wheel, f, overwrite=True)
+                except DatabricksError as err:
+                    logger.warning(f"Uploading wheel files to DBFS failed, perhaps DBFS is protected. {err}")
```

Comment:
Swallow the exception only if user is explicitly requesting for a write-protected dbfs handling

Reply:
@nfx There are no semantics to ask for write protected DBFS. Maybe we can recommend cluster overrides when this fails. I'm not clear on the data structure to know if the overrides are available at the time of this upload. I'll take another look.

Reply:
Addressed this logic. First time through, user will run into ERROR and then instructed to supply cluster overrides.